<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Counter</class>
 <widget class="QWidget" name="Counter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="menu_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rec_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../build-WordCounter-Desktop_Qt_6_5_2_MinGW_64_bit-Debug/debug/qmake_qmake_qm_files.qrc">
     <normaloff>:/i18n/WordCounter_fr_FR.qm</normaloff>:/i18n/WordCounter_fr_FR.qm</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="counter_label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>0</y>
     <width>76</width>
     <height>78</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="go-home">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../build-WordCounter-Desktop_Qt_6_5_2_MinGW_64_bit-Debug/debug/qmake_qmake_qm_files.qrc"/>
 </resources>
 <connections/>
</ui>
